elyra/pipeline/component.py

```diff
@@ -242,7 +242,7 @@ def type(self) -> str:
         return self.type
 
     @abstractmethod
-    def read_component_definition(self, component_id: str, location: str) -> str:
+    def read_component_definition(self, component_id: str, location: str, runtime_type: str) -> str:
```

Comment:
The reader is agnostic of the runtime, so this parameter should not be added.

Reply:
Woops, looks like I kept this in accidentally. Pushing fix now